js/modules/pattern-fill.src.js

```diff
@@ -58,6 +58,11 @@ import H from '../parts/Globals.js';
 * @name Highcharts.PatternOptionsObject#path
 * @type {string|Highcharts.SVGAttributes}
 */ /**
+* SVG patternTransform to apply to the entire pattern.
+* @name Highcharts.PatternOptionsObject#patternTransform
+* @type {string}
+* @see [patternTransform demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/series/pattern-fill-transform)
```

Comment:
I'm just curious: why `@see`, not `@sample`?

Reply:
`@sample` is a custom tag that unfortunately doesn't work for interface doclets..

Reply:
Ah, thanks! 👍